apps/table-frontend/src/app/rated-paragraphs/.gitignore

```diff
@@ -0,0 +1 @@
+*.json
```

Comment:
How come you're gitignoring JSON?  Isn't that your db system?

Reply:
Oh yeah, woops. I got frightened by the 48 added files